FnCAS/fncas/base.h

```diff
@@ -64,7 +64,23 @@ enum class type_t : uint8_t { variable, value, operation, function };
 enum class operation_t : uint8_t { add, subtract, multiply, divide, end };
 enum class function_t : uint8_t { sqr, sqrt, exp, log, sin, cos, tan, asin, acos, atan, unit_step, ramp, end };
```

Comment:
Just realized that there are two different `function_t`-s: this one, and the one defined in `node.h` :-/
I also think that it's not correct to use `_t` notation for `enum class`-es, since they are not type aliases. What do you think about using the following names for the three declarations above: `Operand`, `MathOperation`, `MathFunction`?